File: completeSentence.java
/**
 * completeSentence
 */

 import java.util.Scanner;
public class completeSentence {

    
    public static void main(String[] args) {
        
        Scanner scanner= new Scanner (System.in);

        int n= scanner.nextInt();
        scanner.nextLine();

        for (int i=0;i<n;i++){

            String entrada= scanner.nextLine();
            boolean[] alfabeto= new boolean[26];
            int contador=0;
            for (int j=0;j<entrada.length();j++){

                char caractereAtual= entrada.charAt(j);
                

                if (Character.isLetter(caractereAtual)){

                    int indice= Character.toLowerCase(caractereAtual) -'a';

                    if (alfabeto[indice]==false){

                        alfabeto[indice]= true;
                        contador++;
                    }
                }
            }
                System.out.println(contador);
            }
        }
    }

File: completesentence.c
#include <stdio.h>
#include <string.h>
#include <ctype.h>

int main(){

    int n;

    scanf ("%d", &n);

    for (int i=0;i<n;i++){

        char entrada[200];

        scanf (" %[^\n]", entrada);
        int alfabeto[26];
        int contador=0;
        //INICIALIZAR TODO O ARRAY COM 0.
        for (int z=0;z<26;z++){

            alfabeto[z]= 0;
        }
        for (int j=0;j<strlen(entrada);j++){

            char caractereAtual= entrada[j];
            if (isalpha(caractereAtual)){

                int indice= tolower(caractereAtual) - 'a';

                if (alfabeto[indice]==0){

                    alfabeto[indice]
                    =1;
                    contador++;
                }
            }

        }

        printf ("%d\n", contador);
    }

    return 0;
}